doc/help.rst

```diff
@@ -37,6 +53,23 @@ This can be useful when docstrings are not available or insufficient to document
 
 |
 
+===========================
+Getting help through hovers
+===========================
+
+You can also get help from objects through hovers, which are shown over any method, function or class when placing the cursor on top of it. Clicking the hover will open the information of the object in the Help panel.
```

Comment:
```suggestion
You can also get summary help for objects by hovering over them in the :guilabel:`Editor`.
Clicking the hover popup will open the full documentation in the :guilabel:`Help` pane.
```

* Fix grammar
* Remove a lot of redundancy/repetition
* Improve clarity
* Correct formatting per reST standards and style guide
* Use appropriate terminology (pane vs. panel)